webapp/components/CategoriesSelect/CategoriesSelect.spec.js

```diff
@@ -35,14 +37,19 @@ describe('CategoriesSelect.vue', () => {
     },
   ]
   beforeEach(() => {
+    provide = {
+      $parentForm: {
+        update: jest.fn(),
+      },
+    }
```

Comment:
This is how we can implement dedicated form input components, e.g. the `CategoriesSelect` component. @mattwr18

Reply:
looks pretty cool @roschaefer